### PR TITLE
Fix VSBC RTL views

### LINKED ISSUE #30660: Some charts are not rendering correctly in RTL
### LTR 

image

### RTL

image

Notice that the order of axis ticks is being reversed from LTR to RTL, but this is not happening with the bars.

### PR DESCRIPTION
**Pull request by Shubhabrata08**
## Previous Behavior

RTL views were not working properly for VerticalStackedBarChart, specifically for DateAxis and StringAxis

## New Behavior

RTL views have been fixed for VerticalStackedBarChart.
Here's a demo:
https://jam.dev/c/f1920ccc-c0fe-4d5b-b013-dc57372411f3

## Related Issue(s)

- Fixes #30660

### CODE REVIEW

**Comment by AtishayMsft** [comment]
Thanks for the great demo above.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/107d301f69bed534d533e18251407d121aad6dd5'>107d301f69bed534d533e18251407d121aad6dd5</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-df9f50b3-f2d6-49d3-84c7-77b1ed032133.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-df9f50b3-f2d6-49d3-84c7-77b1ed032133.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30691&commitId=d44d07cb3acb261fa55019e4a940bbd673e5bac7&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Stacked - Date Axis - RTL.default.chromium.png|16055| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by AtishayMsft — APPROVED** [review]

File: change/@fluentui-react-charting-df9f50b3-f2d6-49d3-84c7-77b1ed032133.json (hunk 1 above)
Comment (AtishayMsft) [reply change/@fluentui-react-charting-df9f50b3-f2d6-49d3-84c7-77b1ed032133.json:1]: This PR fixes the issue being observed in the baseline screenshot

**Comment by AtishayMsft** [comment]
Some issue is being reported in installing dependencies. Could you merge latest master.

**Pull request merged**